documentation/setup/prerequisites.md

```diff
@@ -7,7 +7,7 @@ description: "Prerequisites"
 ## Requirements
 
 :::caution
-If **all** of the points in the list below does not match you, **do not** try to setup/use HACS.
+**DO NOT** try to setup/use HACS unless you agree with **all** of the following! 
 :::

```

Comment:
Just remove the note, requirements are requirements.
most folks do not read or care about these anyway